src/Content/Widget.php

```diff
@@ -45,7 +45,7 @@ class Widget
 	 * @return string
 	 * @throws \Friendica\Network\HTTPException\InternalServerErrorException
 	 */
-	public static function follow($value = "")
+	public static function follow(string $value = ""): string
```

Comment:
While you're at it, please normalize quotes in the lines you edited.
```suggestion
	public static function follow(string $value = ''): string
```